src/request.rs

```diff
@@ -26,6 +26,32 @@ pub struct Request<State> {
     pub(crate) route_params: Vec<Params>,
 }
 
+#[derive(Debug)]
+pub enum ParamError<T: FromStr> {
```

Comment:
what is the `use` path to get to this from a tide app if someone needs to match on it? that probably needs to be specified in lib.rs

Reply:
Yes, I added it in lib.rs [here](https://github.com/http-rs/tide/pull/615/files#diff-b4aea3e418ccdb71239b96952d9cddb6R219), thanks!